pixl_ehr/tests/test_processing.py

```diff
@@ -186,11 +257,81 @@ async def test_message_processing() -> None:
 
         assert value == expected_value
 
-    anon_row = pixl_db.execute_query_string(
-        "select * from emap_data.ehr_anon where gcs = %s", [gcs]
+    anon_all_rows = pixl_db.execute_query_string(
+        "select mrn, accession_number, xray_report from emap_data.ehr_anon where gcs = %s", [gcs]
     )
-    anon_mrn, anon_accession_number = anon_row[:2]
+    assert len(anon_all_rows) == 1
+    anon_row = anon_all_rows[0]
+    anon_mrn, anon_accession_number, anon_report_text = anon_row
     assert anon_mrn != mrn
-    assert anon_accession_number != accession_number
+    assert anon_accession_number != message.accession_number
+
+    # No deidentification performed so we expect the report to stay identical
+    assert report_text == anon_report_text
```

Comment:
yeah maybe we should make the mock cogstack add on something like `**DE-IDENTIFIED**` to the end of the string so that we know it has been processed in tests

Reply:
I agree but can we do it in a separate PR? There are quite a few other changes I want to make to testing that could be done together as a separate piece of work.
I would also suggest that we monkeypatch `deidentify_text` during unit tests so we don't need the fake cogstack service except for in integration/system tests, but then it might not be the only thing that needs monkeypatching...

Reply:
Oops, sorry @jeremyestein I already pushed this into this PR before seeing your comment

Reply:
No problem